content/desktop/synchronized-file-sharing.md

```diff
@@ -1,60 +1,41 @@
 ---
-title: Synchronized file sharing
-description: Get started with Synchronized file sharing on Docker Desktop.
-sitemap: false
+title: Synchronized File Shares
+description: Get started with Synchronized File Shares on Docker Desktop.
+keyword: mutagen, file sharing, docker desktop, bind mounts
 ---
 
-> **Early Access**
+> **Note**
 >
-> Synchronized file sharing is an early-access feature. 
->
-> If you would like to be considered for Synchronized file sharing testing, you can
-> [sign up for the early access program](https://www.docker.com/docker-desktop-preview-program/).
-{ .restricted }
+> Synchronized File Shares is available for customers with a Docker subscription. 
 
-Synchronized file sharing is an alternative file sharing mechanism powered by [Mutagen](https://mutagen.io/). It provides fast and flexible host-to-VM file sharing by replacing bind mounts with synchronized filesystem caches. 
+Synchronized File Shares is an alternative file sharing mechanism powered by [Mutagen](https://mutagen.io/). It provides fast and flexible host-to-VM file sharing by replacing bind mounts with synchronized filesystem caches. 
```

Comment:
I think I would opt to remove it; I think we want to avoid any confusion if people start veering into the Mutagen docs.  I'd just say "Synchronized File Shares is an alternative file sharing mechanism that provides fast and flexible host-to-VM ..."